src/components/ListPaginationBottom.tsx

```diff
@@ -57,7 +59,10 @@ export const ListPaginationBottom = ({
         aria-label={paginationAriaLabel}
         page={page}
         count={pages}
-        onChange={(_, newPage) => onPageChange(newPage)}
+        onChange={(_, newPage) => {
+          scrollToDivRef?.current?.scrollIntoView({ behavior: 'auto', block: 'start' });
```

Comment:
alltid definert ifølge typen?
```suggestion
          scrollToDivRef.current?.scrollIntoView({ behavior: 'auto', block: 'start' });
```